### DISCUSSION TITLE
Are there any plans of integrating OAuth for https?

### DISCUSSION THREAD

**Discussion by domportera** [Murmele/Gittyup#220] [General]
just wondering :)

**Comment by exactly-one-kas** [comment]
Do you mean OAuth for remote accounts or for GIT remotes?

**Reply by domportera** [reply]
for remote accounts, like git, bitbucket, etc

**Reply by exactly-one-kas** [reply]
There's currently no specific plans to implement this, but you can use a personal access token as the password to log in (be sure to copy the token without any whitespaces)

**Reply by exactly-one-kas** [reply]
I created #226, so you can check for progress there. However, I can't guarantee that this will be implemented in the near future

**Reply by AMDphreak** [reply]
> There's currently no specific plans to implement this, but you can use a personal access token as the password to log in (be sure to copy the token without any whitespaces)

I would advise against this, because GitHub's tokens implementations are confusing. The old one does everything one expects by default but is not advisable, while the new one (fine-grained) requires you to ~just know~ that you need to specify the permissions, because GitHub, in its wisdom, decided not to select a reasonable default set of permissions.

Want to know why I'm here, replying to this? Because I had to google this to double check to see if I have all of the necessary permissions. As of writing this, I still don't know, because this specific information is not readily available. @Murmele  Don't cause your users to need to go do research just to login to GitHub. That's lame.
